### PR TITLE
Tooltip picture elements

### PR DESCRIPTION
**Pull request by yosilevy**
Added a way to suppress the UGLY tooltips in picture-elements.

Also added localization for the existing tooltips.

The docs on the current ability to specify a tooltip is lacking (only there for icon element). 
I will fix this and also add docs on the new noTitle option.

After this PR is approved...

### CODE REVIEW

File: src/panels/lovelace/common/compute-tooltip.ts [hunk 1]
```diff
@@ -6,11 +6,16 @@ import { ActionConfig } from "../../../data/lovelace";
 interface Config extends LovelaceElementConfig {
   entity?: string;
   title?: string;
+  noTitle?: boolean;
```
Comment (iantrich) [inline src/panels/lovelace/common/compute-tooltip.ts]: camelcase is inconsistent with how we define all other configuration options. I think `show_tooltip` is a better descriptor.

Comment (yosilevy) [reply src/panels/lovelace/common/compute-tooltip.ts]: Ooops.
show is the default behavior so I don't want to change that. I changed it to hide_title.

Comment (iantrich) [reply src/panels/lovelace/common/compute-tooltip.ts]: `show` is the common phrase currently used for things like this. e.g. https://www.home-assistant.io/lovelace/picture-entity/#show_name

Comment (yosilevy) [reply src/panels/lovelace/common/compute-tooltip.ts]: Done

Comment (balloob) [reply src/panels/lovelace/common/compute-tooltip.ts]: We should just allow setting `title` to `null` to indicate no title should be shown.

File: src/panels/lovelace/common/compute-tooltip.ts [hunk 2]
```diff
@@ -48,20 +56,31 @@ function computeActionTooltip(
...
+    (isHold
+      ? hass.localize("ui.panel.lovelace.cards.picture-elements.hold")
+      : hass.localize("ui.panel.lovelace.cards.picture-elements.tap")) + " ";
 
   switch (config.action) {
     case "navigate":
-      tooltip += `Navigate to ${config.navigation_path}`;
+      tooltip += `${hass.localize(
+        "ui.panel.lovelace.cards.picture-elements.navigate_to"
+      )} ${config.navigation_path}`;
```
Comment (balloob) [inline src/panels/lovelace/common/compute-tooltip.ts]: Don't concatenate strings. Instead the localization string should have a placeholder.

File: src/translations/en.json [hunk 3]
```diff
@@ -916,6 +916,14 @@
             "checked_items": "Checked items",
             "clear_items": "Clear checked items",
             "add_item": "Add item"
+          },
+          "picture-elements": {
+            "hold": "Hold:",
+            "tap": "Tap:",
+            "navigate_to": "Navigate to",
```
Comment (balloob) [inline src/translations/en.json]:
```suggestion
            "navigate_to": "Navigate to {location}",
```

Comment (yosilevy) [reply src/translations/en.json]: done

File: src/panels/lovelace/common/compute-tooltip.ts (hunk 1 above)
Comment (yosilevy) [reply src/panels/lovelace/common/compute-tooltip.ts]: Didn't think that was such good practice. Anyway it's easier so done.

**Review by balloob — APPROVED** [review]

**Pull request merged**